src/Views/PartitioningView.vala

```diff
@@ -258,19 +264,33 @@ public class Installer.PartitioningView : AbstractInstallerView {
         }
     }
 
-    private void set_mount (Mount mount) {
+    private string? set_mount (Mount mount) {
```

Comment:
To me all the returned strings look like errors,
better do a `private void set_mount (Mount mount) throws GLib.Error` and then `throw new GLib.IOError.FAILED (_("blah"));`